src/ui/components/pages/contacts/accreditationsDetail/icons.js

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { useAPI } from "core/hooks";
+import { Chip, Divider, Typography } from "@mui/material";
+
+export const AccreditationsIcons = ({ idec }) => {
+  const { getContactAccreditations } = useAPI();
+
+  const [pageState, setPageState] = useState({
+    isLoading: false,
+    data: [],
+  });
+
+  useEffect(() => {
+    setPageState(old => ({ ...old, isLoading: true }));
+    (async () => {
+      const { data } = await getContactAccreditations(idec);
+      setPageState(old => ({
+        ...old,
+        isLoading: false,
+        data: data,
+      }));
+    })();
+  }, [idec]);
+
+  return (
+    <>
+      {pageState.data.map(acc => (
+        <>
```

Comment:
React fragment (`<>`) is useless here, you can remove it.

When you generate a list of item, React want to you to add `key` props to elements.
For exemple, you can do :
```js
{pageState.data.map(acc => (
        <Chip
          key={`${JSON.stringify(acc)}-chip`}
          ...>
          {...}
         </Chip>
))}
```